app/services/balance_migration.rb

```diff
@@ -55,20 +55,20 @@ def aggregate
   # this method reeks of :reek:DuplicateMethodCall, :reek:TooManyStatements
   def success?
     User.find_each do |user|
-      old_b = old_total_balance(user)
-      new_b = new_total_balance(user)
+      old_b = old_total_debt(user)
+      new_b = new_total_debt(user)
```

Comment:
Since the entity you operate on changed from `Balance` to `Debt`, maybe the variables should be also renamed from `old_b` to `old_d`?